keccak-hash/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "keccak-hash"
-version = "0.2.0"
+version = "0.2.1"
```

I believe that's a breaking change for `keccak-hash`
```suggestion
version = "0.3.0"
```

---

Probaly if it reexport or type alias, I did not thought about checking that for other crates either.